### ISSUE TITLE
Menu na tela de entrada - ambiente de homologação

### ISSUE THREAD

**Issue by RochaCarla** [fga-gpp-mds/2017.2-SiGI-Op_API#148]
Na tela inicial da aplicação deveria aparecer o menu da mesma?

**Comment by lopes05** [comment]
Não. Essa sprin foi aberta uma issue dedicada a arrumar tal problema, falta apenas realizar o merge e subir para a versão de produção.

**Comment by mateus-andrade** [comment]
Esse bug foi corrigido no PR [#41](https://github.com/fga-gpp-mds/2017.2-SiGI-Op/pull/41) juntamente com outro, vamos atualizar o ambiente de homologação.
